src/node.h

```diff
@@ -264,6 +264,7 @@ NODE_EXTERN Environment* CreateEnvironment(IsolateData* isolate_data,
 
 NODE_EXTERN void LoadEnvironment(Environment* env);
 NODE_EXTERN void FreeEnvironment(Environment* env);
+NODE_EXTERN Environment* GetCurrentEnvironment(v8::Local<v8::Context> context);
```

Comment:
Can you add a comment similar to the one for `GetCurrentEventLoop()`?

Reply:
done!